src/routes/api/+server.js

```diff
@@ -6,25 +6,25 @@ export async function POST({ request }) {
 
     // Check if the App exists in the database based on the appId
     let existingApp = await prisma.app.findUnique({
-        where: { appId: data.appId }
+        where: { appId: data[0].appId }
```

Comment:
The posted JSON (from https://docs.google.com/document/d/1EAP4l3ZuCab-pzrDrm9bV6m4KhTmd6OG-RroSohoI1Q/edit) is a single object, not an array of objects.

Reply:
I think it expects the JSON to be an array of objects. For me when I tried accessing data.anyOfTheAttributes, it came up as undefined, but using data[0].attribute worked

Reply:
What were you POSTing to the API? It should look like:

```
{
	"appId" : "CE2679C5-1078-4820-814D-E0D0310A30D9",
	"appIcon" : "https://app.scriptoria.io/api/products/CE2679C5-1078-4820-814D-E0D0310A30D9/files/published/app-icon.png",
	"googlePlayUrl" : "https://play.google.com/store/apps/details?id=org.example.test",
	"listings" : [ {
"lang" : "en-US", 
"title" : "The App Name From Google Play",
"shortDescription" : "The Short Description from Google Play",
"fullDescription" : "The Full Description from Google Play"
} ,
 	{ 
"lang" : "es-419", 
  		"title" : "El nombre de la aplicación de Google Play",
    		"shortDescription" : "La breve descripción de Google Play",
    		"fullDescription" : "La descripción completa de Google Play"
  	}]
}
```

Reply:
That is the same thing that I was POSTing